### PR TITLE
[ASM] - Expander - Lowercase GridField Keys

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-3672

## Description
Lowercase GridField keys for layouts

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , can you please review?

**Comment by johnnywilkes** [comment]
@capanw / @BigEasyJ , please review

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md [hunk 1]
```diff
@@ -38,12 +38,12 @@ This playbook does not use any integrations.
...
-| AcceptedRiskProjects | Comma-separated list of projects numbers that are considered an acceptable risk and that should be closed.  For example, a list of GCP projects and AWS accounts. |  | Optional |
-| AcceptedRiskOther | Comma-separated list of other items that are considered an acceptable risk and that should be closed. For example, a list of folder numbers in GCP and subscription IDs in Azure. |  | Optional |
-| JiraProjectKey | The Jira project key to associate the issue. | XPANSE | Required |
+| AcceptedRiskDs | Comma separated list of instance/VM IDs that are considered accepted risk and that should be closed. |  | Optional |
+| AcceptedRiskProjects | Comma separated list of projects numbers that are considered accepted risk and that should be closed.  For example, the could be list of GCP projects and AWS accounts. |  | Optional |
+| AcceptedRiskOther | Comma separated list of other items that are considered accepted risk and that should be closed. For example, the could be list of folders numbers in GCP and subscription IDs in Azure. |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md:45]:
```suggestion
| AcceptedRiskOther | Comma-separated list of other items that are considered an accepted risk and that should be closed. For example, a list of folders numbers in GCP and subscription IDs in Azure. |  | Optional |
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_30.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+##### Cortex ASM - Remediation
+
+Updated the playbook to use grid field with lowercase keys.
+
+#### Scripts
+
+##### RemediationPathRuleEvaluation
+
+- Updated the script to use grid field with lowercase keys.
+- Updated the Docker image
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_30.md:89]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.12.68300*.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_30.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+##### RankServiceOwners
+
+- Updated the script to use grid field with lowercase keys.
+- Updated the Docker image
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_30.md:94]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.12.68300*.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_30.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+##### GenerateASMReport
+
+- Updated the script to use grid field with lowercase keys.
+- Updated the Docker image
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_30.md:98]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.12.68300*.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_30.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+##### InferWhetherServiceIsDev
+
+- Updated the script to use grid field with lowercase keys.
+- Updated the Docker image
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_30.md:103]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.12.68300*.
```

**Comment by ShirleyDenkberg** [comment]
@efelmandar @mmhw Doc review completed.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md:1]: Alphabetical order for list please.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 6]
```diff
@@ -3215,17 +3215,17 @@ inputs:
...
   value: {}
   required: false
-  description: Comma separated list of instance/VM IDs that are considered accepted risk and that should be closed.
+  description: Comma-separated list of instance/VM IDs that are considered an accepted risk and that should be closed.
   playbookInputQuery:
 - key: AcceptedRiskProjects
   value: {}
   required: false
-  description: Comma separated list of projects numbers that are considered accepted risk and that should be closed.  For example, the could be list of GCP projects and AWS accounts.
+  description: Comma-separated list of projects numbers that are considered an accepted risk and that should be closed.  For example, a list of GCP projects and AWS accounts.
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:3223]:
```suggestion
  description: Comma-separated list of projects numbers that are considered an accepted risk and should be closed.  For example, a list of GCP projects and AWS accounts.
```

**Review by capanw — APPROVED** [review]

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 7]
```diff
@@ -3215,17 +3215,17 @@ inputs:
...
   playbookInputQuery:
 - key: AcceptedRiskOther
   value: {}
   required: false
-  description: Comma separated list of other items that are considered accepted risk and that should be closed. For example, the could be list of folders numbers in GCP and subscription IDs in Azure.
+  description: Comma-separated list of other items that are considered an accepted risk and that should be closed. For example, a list of folders numbers in GCP and subscription IDs in Azure.
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:3228]:
```suggestion
  description: Comma-separated list of other items that are considered an accepted risk and should be closed. For example, a list of folders numbers in GCP and subscription IDs in Azure.
```

**Review by BigEasyJ — APPROVED** [review]
Just small changes, that don't block merging.

**Comment by johnnywilkes** [comment]
@mmhw / @efelmandar , this is approved from our side.  Please merge when possible

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md:1]: This was generated with SDK, want to put in a enhancement to have them set lists in alphabetical order?

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml (hunk 6 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:3223]: Shirley reviewed and plan to keep as-is

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml (hunk 7 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:3228]: Shirley reviewed and plan to keep as-is

**Review by mmhw — APPROVED** [review]
Nice!

**Pull request merged**